bridges/darwinia-ethereum/bridge/src/service/affirm/handler.rs

```diff
@@ -63,26 +61,26 @@ impl AffirmHandler {
 
         tracing::info!(
             target: "darwinia-ethereum",
-            "✨ SERVICE STARTED: ETHEREUM <> DARWINIA RELAY"
+            "[ethereum] [affirm] ✨ AFFIRM RECRATED: ETHEREUM <> DARWINIA RELAY"
         );
         Ok(AffirmHandler {
             microkv,
             sender_to_extrinsics,
-            darwinia,
+            client,
             shadow,
         })
     }
 }
 
 impl AffirmHandler {
     pub async fn affirm(&mut self) -> color_eyre::Result<()> {
-        let last_confirmed = self.darwinia.last_confirmed().await?;
+        let last_confirmed = self.client.ethereum().last_confirmed().await?;
```

Comment:
To double check

Reply:
before

https://github.com/darwinia-network/bridger/blob/c8c1a5babff6cc3d741fa4b26ce6d5b3e39c1ac6/bridges/darwinia-ethereum/client-darwinia/src/frame/ethereum/relay.rs#L162-L168

https://github.com/darwinia-network/bridger/blob/c8c1a5babff6cc3d741fa4b26ce6d5b3e39c1ac6/bridges/darwinia-ethereum/client-darwinia/src/from_ethereum/api.rs#L197-L206

after

https://github.com/darwinia-network/bridger/blob/2e4afda99f8bd9520f66b2714ccb1efb03e8a8f1/bridges/darwinia-ethereum/client-darwinia/src/ethereum/api.rs#L31-L41